app/src/main/java/com/whereuat/whereu/db/WhereuatContract.java

```diff
@@ -0,0 +1,70 @@
+package com.whereuat.whereu.db;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.provider.BaseColumns;
+import android.util.Log;
+import android.util.Pair;
+
+import java.security.Key;
+
+/**
+ * Created by kangp3 on 4/3/16.
+ */
+public final class WhereuatContract {
+    private static final String TAG = "WuaContract";
+
+    public WhereuatContract() {}
+
+    public static abstract class ContactEntry implements BaseColumns {
+        public static final String TABLE_NAME = "contacts";
+        public static final String COLUMN_NAME_NAME = "name";
+        public static final String COLUMN_NAME_PHONE = "phone";
+        public static final String COLUMN_NAME_AUTOSHARE = "autoshare";
+    }
+
+    public static abstract class KeyLocationEntry implements BaseColumns {
+        public static final String TABLE_NAME = "key_locations";
+        public static final String COLUMN_NAME_NAME = "name";
+        public static final String COLUMN_NAME_LATITUDE = "lat";
+        public static final String COLUMN_NAME_LONGITUDE = "lng";
+    }
+
+    @SafeVarargs
+    public static String createTable(String table_name, Pair<String, String>... columns) {
+        String sql_string = "CREATE TABLE " + table_name;
+        if (columns.length == 0) {
```

Comment:
If this `if` happens then won't we return `"CREATE TABLE table_name`? That's no good. I think this should return an empty string or null. Also, put it at the top of the function so we don't unnecessarily make a String.